Staging_Dev/20190401T202900Z_welcome_mail.sql

```diff
@@ -0,0 +1,46 @@
+USE [perpetuumsa]
+GO
+
```

Comment:
Drop if exists before any new table creation in patches - will allow it to be re-run in case of other issues and is generally safer (@Bler for details)